RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate methods/loadMessagesForRoom to Typescript

## Proposed changes
Migrate methods/loadMessagesForRoom to typescript

## Issue(s)	
N/A

## How to test or reproduce
 * Open a chat,  messages should load correctly.
 * Clear the app cache, open a chat, messages should load correctly.

## Screenshots
N/A

## Types of changes




## Checklist



## Further comments
N/A
